packages/core/src/browser/window/default-secondary-window-service.ts

```diff
@@ -100,7 +105,13 @@ export class DefaultSecondaryWindowService implements SecondaryWindowService {
     }
 
     protected doCreateSecondaryWindow(widget: ExtractableWidget, shell: ApplicationShell): Window | undefined {
-        const newWindow = window.open(DefaultSecondaryWindowService.SECONDARY_WINDOW_URL, this.nextWindowId(), 'popup') ?? undefined;
+        let options;
+        const [height, width, left, top] = this.findSecondaryWindowCoordinates(widget);
+        options = `popup=1,width=${width},height=${height},left=${left},top=${top}`;
+        if (this.preferenceService.get('window.secondaryWindowAlwaysOnTop')) {
+            options += ',alwaysOnTop=true';
```

Comment:
`alwaysOnTop` makes the window to be on top of all windows, at least on Ubuntu with the electron application. 
When I move a different application to this screen, e.g. the browser, the popup will also be on top of the browser. I would have expected the popup to behave like a modal dialog without disabling interaction with the main application. 

Is this true on your OS as well? Can this be changed? If not maybe the default should be `false` instead. What do you think?

In the browser application this does not seems to be supported as it opens a new browser window for me.

Reply:
I agree, I set the default to `false` and updated the preference description to better explain the behavior.